src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -1677,8 +1681,9 @@ static freeze_result freeze_epilog(JavaThread* thread, ContinuationWrapper& cont
   return freeze_epilog(cont);
 }
 
-static freeze_result preempt_epilog(ContinuationWrapper& cont, freeze_result res, frame& old_last_frame) {
+static freeze_result preempt_epilog(JavaThread* thread, ContinuationWrapper& cont, freeze_result res, frame& old_last_frame) {
```

Comment:
If `thread` is always the current thread, please name it `current`.

Reply:
> If `thread` is always the current thread, please name it `current`.

It is always the current thread (freezing is always done by the thread that owns the frames). Please do the same for `freeze_epilog` just above `preempt_epilog`.